src/sentry/static/sentry/app/components/checkboxFancy.jsx

```diff
@@ -11,7 +11,7 @@ class CheckboxFancy extends React.Component {
 
   static defaultProps = {
     checked: false,
-    size: '18px',
+    size: '16px',
```

Comment:
Is this checkbox used outside of the global selection header?

Reply:
yeah. I'm gonna verify the other instances.

Reply:
actually...it isn't.